spanner/client.go

```diff
@@ -937,12 +946,17 @@ func (c *Client) Apply(ctx context.Context, ms []*Mutation, opts ...ApplyOption)
 	defer func() { trace.EndSpan(ctx, err) }()
 
 	if !ao.atLeastOnce {
+		var commitOptions CommitOptions
+		if ao.commitOptions != nil {
+			commitOptions = *ao.commitOptions
+		}
```

Comment:
I don't quite understand why this is different depending on whether `ao.atLeastOnce` is true or not.